### ISSUE TITLE
How can I save GradientBoostingSurvivalAnalysis model without pickle?

### ISSUE THREAD

**Issue by KOLANICH** [sebp/scikit-survival#19]

**Comment by sebp** [comment]
I don't think there is easy way to do this, because we are sub-classing scikit-learn's implementation of gradient boosting. The fitted model has an `estimators_` attribute, which is a numpy array of `sklearn.tree.DecisionTreeRegressor`, which contains a `tree_` attribute containing the underlying `Tree` object and therefore the actual splits.
